projects/plugins/search/CHANGELOG.md

```diff
@@ -5,6 +5,19 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [1.1.0-beta] - 2022-07-29
+### Changed
+- E2E tests: bump @playwright/test and allure-playwright versions
+- Renaming master to trunk.
+- Renaming `master` references to `trunk`
+- Updated package dependencies.
+
+### Fixed
+- E2E tests: fixed pretest cleanup script not running
+- Search: update plugin README
+
 ## 1.0.0 - 2022-05-30
 ### Added
 - Initial release.
+
+[1.0.1-beta]: https://github.com/Automattic/jetpack-search-plugin/compare/v1.0.0...v1.0.1-beta
```

Comment:
```suggestion
[1.1.0-beta]: https://github.com/Automattic/jetpack-search-plugin/compare/v1.0.0...v1.0.1-beta
```

Reply:
Looks like I need to update the version number in the compare URL too.